### ISSUE TITLE
SickRage, CouchPotato not detected 2~ months after install (for updating, password change, etc)

### ISSUE THREAD

**Issue by hjdarnel** [htpcBeginner/AtoMiC-ToolKit#171]
Hi, a couple months back I installed my HTPC setup, on my Raspberry Pi running OSMC (Raspbian Jessie). Yesterday, I was moving house and went to change passwords, etc, and running `sudo bash ./setup.sh` only lists NZBGet. All three of these utils were installed using AtoMiC, so I'm unsure what to do to get these listed again. Also - these utils were/are running properly.

**Comment by TommyE123** [comment]
hi, 
Have you checked your running the latest version of the toolkit? 
Can you try using the maintenance menu - Update AtoMiC ToolKit

Let me know how you get on and I'll try and hep you further if needed.

Thanks
Tom

**Comment by hjdarnel** [comment]
> Have you checked you're running the latest version of the toolkit?

Yes, I've pulled master. 

> Can you try using the maintenance menu - Update AtoMiC ToolKit

When I run ./setup.sh, it doesn't give me the option to run any maintenance, it just sees that OSMC is installed and then exits.

**Comment by TommyE123** [comment]
Hi @hjdarnel 

Sorry I've not got back to you regarding this problem, I've been at a bit of a lose as to how to fix this. However I've recently made some changes to the toolkit which might help with your issues. If you still want to give it another go please do. Or if you no longer require any help, can you close this issue for me?

Sorry again I've been unable to help further. If you still have issues and want to try and get it working can you contact me on [Gitter](https://gitter.im/Tommye123).

Tom

**Comment by hjdarnel** [comment]
Hi Tom, I will try to pull `master` and update everything here in the next few days when I get some time, and I'll try to contact you if you think it'd be helpful. As of the last time I tried to add a service (about 2 weeks ago) this issue was still evident, so I'll keep it open for now.

**Comment by TommyE123** [comment]
Hi @hjdarnel

I recently came across someone else having issues similar to you. I think this might have now been resolved by a missing dependency for the toolkit which has now been added. Can you pull the master branch again?

If you still have issues and want to try and get it working can you contact me on [Gitter](https://gitter.im)

Thanks
Tom

**Comment by hjdarnel** [comment]
Hi @TommyE123, thanks for following up again. I've pulled and run `-t` flag to update.

Running `sudo bash setup.sh` with no options brings up the UI where I can view different services and maintain them. I have couchpotato, sickrage, plex, and NZBget running on my server. All these services *except NZBget* show as installed when I try to view access details. I get the following message:

`Error! /opt/nzbget not found. NZBGet not installed or incompatible installation.` (indeed, it's not there)

I **believe** NZBGet was installed with Atomic toolkit. My nzbget folder is under my home directory. Would Atomic have ever put it there on install? If not, it's possible I had installed it myself.
